uitest/README.md

```diff
@@ -0,0 +1,57 @@
+## Dev Home UI Testing
+### How to run UI tests
+1. Open Dev Home solution in Visual Studio
+2. Install the version of Dev Home you want to test.
+   - Note: For testing a developer build of Dev Home, build and deploy the solution from Visual Studio.
+3. Install [Windows Application Driver](https://github.com/microsoft/WinAppDriver/releases/download/v1.2.99/WindowsApplicationDriver-1.2.99-win-x64.exe)
```

Comment:
C'mon! You should have included the winget command ;)
`winget install Microsoft.WindowsApplicationDriver`
(Though it's outdated and maybe not the supported way to install...)